ImperatorToCk3/Source/Imperator/ImperatorWorld.cpp

```diff
@@ -0,0 +1,108 @@
+#include "ImperatorWorld.h"
+#include "../Common/CommonFunctions.h"
+#include "../Common/Version.h"
+#include "../Configuration/Configuration.h"
+#include "Date.h"
+#include "Log.h"
+#include "OSCompatibilityLayer.h"
+#include "ParserHelpers.h"
+#include <ZipFile.h>
+#include <filesystem>
+#include <fstream>
+
+namespace fs = std::filesystem;
+
+ImperatorWorld::World::World(const Configuration& theConfiguration)
+{
+	LOG(LogLevel::Info) << "*** Hello Imperator, Roma Invicta! ***";
+	registerRegex("\\bSAV\\w*\\b", [](const std::string& unused, std::istream& theStream) {});
```

Comment:
You can use a more modern approach to these regex strings that makes them more readable:
registerRegex(R"(\bSAV\w*\b)", []...

Reply:
Oh yeah I'm familiar with raw from Python, nice to know C++ has it too.